ShapesProject/Project1/Main.cpp

```diff
@@ -0,0 +1,171 @@
+#include <iostream>
+#include "FShape.h"
+#include "FCircle.h"
+#include "FSquare.h"
+#include "TStaticArray.h"
+#include "TDynamicArray.h"
+
+#include "ValidateInput.h"
+
+#define Print(Text) std::cout << Text << std::endl;
+
+//------------------------------------------------------------------------------------------
+
+FCircle* GetCircle()
+{
+	float Radius = GetValidateInput("Enter the radius: ");
+	FCircle* NewCircle = new FCircle(Radius);
+
+	return NewCircle;
+}
+
+//------------------------------------------------------------------------------------------
+
+FSquare* GetSquare()
+{
+	float Size = GetValidateInput("Enter the Size of the edges: ");
+	FSquare* NewSquare = new FSquare(Size);
+	return NewSquare;
+}
+
+//------------------------------------------------------------------------------------------
+
+template<typename T>
+void PrintArray(const T& InArray, int InSize)
+{
+	system("cls");
+	Print("--------------------------Array Content--------------------------");
+	for (int i = 0; i < InSize; i++)
+	{
+		float LoopArea = InArray[i]->GetArea();
```

Comment:
You might be looking for Iteration as a prefix instead of Loop, so IterationArea and IterationPerimeter are a far better fit.